umibukela/views.py

```diff
@@ -46,11 +45,40 @@ def site(request, site_slug):
     })
 
 
+def site_result_hardcoded(request, site_slug, result_id):
+    result_set = CycleResultSet.objects.get(
+        id=result_id,
+        site__slug__exact=site_slug
+    )
+    return render(request, 'site_result_hardcoded.html', {
+        'active_tab': 'sites',
+        'result_set': result_set,
+    })
+
+
 def site_result(request, site_slug, result_id):
-    result_set = CycleResultSet.objects.get(id=result_id, site__slug__exact=site_slug)
+    result_set = CycleResultSet.objects.get(
+        id=result_id,
+        site__slug__exact=site_slug
+    )
+    site_submissions = Submission.objects.filter(cycle_result_set=result_set)
+    site_responses = [submission.answers for submission in site_submissions]
+    site_totals = results.count_submissions(pandas.DataFrame(site_responses))
+    form = result_set.survey.form
+    site_results = results.count_options(
+        pandas.DataFrame(site_responses),
```

Comment:
Can we stash and re-use the dataframe? How expensive is creating a new dataframe?